monkey/infection_monkey/monkey_utils/windows/new_user.py

```diff
@@ -0,0 +1,69 @@
+import logging
```

Comment:
The location of the file is problematic.
`monkey_utils` is intended for source code for C, assembly, batch scripts, or whatever that is not part of the monkey python code.
You can rename this directory if you want (and you'll need to update the monkey readmes accordingly), and use the `monkey_utils` name for files like this.
Just separate the folders 🙃

Reply:
edc2d493